gadgets/top_tcp/README.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: top_tcp
+sidebar_position: 50
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# top_tcp
+
+The top_tcp gadget reports tcp send receive activity by connection.
+
+## Getting started
+
+Running the gadget:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/top_tcp:latest [flags]
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo ig run ghcr.io/inspektor-gadget/gadget/top_tcp:latest [flags]
+        ```
+    </TabItem>
+</Tabs>
+
+## Flags
+
+### `--family`
+
+Trace traffic from the specified IPv family. Can be either 4 or 6. Specifying -1 equals to no filtering
+
+Default value: "-1"
```

Comment:
Did you generate the file manually or did you use README.template? This was moved from README.md to README.mdx but the Makefile still refers to README.md so it seems out of sync.

Reply:
We manually copied the contents from README.md to README.mdx.